nle/scripts/read_tty.py

```diff
@@ -67,13 +67,15 @@ def color(s, value):
                 data = action
                 channel = "->"
 
-            data = re.sub(r"\\x1b\[([0-9];?)*.", lambda m: color(m.group(0), 8), data)
+            data = re.sub(
+                r"\\x1b\[([0-9];?)*.", lambda m: (color(m.group(0), 8), str(data))
```

Comment:
Incidentally, this wasn't quite correct: The lambda shouldn't return a tuple; the `str(data)` should be the third argument of `re.sub`.